pints/_logger.py

```diff
@@ -386,19 +386,22 @@ def log(self, *data):
             column = iter(row)
             formatted_row = []
             for width, dtype, f1, f2 in self._field_formats:
-                if dtype == _FLOAT:
-                    v = next(column)
-                    x = f1.format(v)
-                    if len(x) > width:
-                        x = f2.format(v)
-                    x += ' ' * (width - len(x))
-                elif dtype == _TIME:
-                    x = self._format_time(next(column))
-                elif dtype == _TEXT:
-                    x = str(next(column))[:width]
-                    x += ' ' * (width - len(x))
+                v = next(column)
+                if v is None:
+                    x = ' ' * width
                 else:
```

Comment:
Could be an elif in case we want to save space :D

Reply:
done